rele/client.py

```diff
@@ -60,7 +60,8 @@ def create_subscription(self, subscription, topic):
         subscription_path = self._client.subscription_path(
             self._gc_project_id, subscription
         )
-        topic_path = self._client.topic_path(self._gc_project_id, topic)
+        publisher = pubsub_v1.PublisherClient()
+        topic_path = publisher.topic_path(self._gc_project_id, topic)
```

Comment:
I wonder if its still possible to do this with the SubscriptionClient following the [docs here](https://googleapis.dev/python/pubsub/latest/subscriber/index.html#creating-a-subscription). 

If not, then we have to be very careful about memory leaks, as if IIRC, the publisher class is the one that gave us trouble.

Reply:
I'm not so sure about this now. I can't seem to find any `topic_path` method for the subscriber in the pubsub project, only the `PublisherClient` class  has it. Perhaps that section of the docs was not updated?

Reply:
I'm not so sure about this now. I can't seem to find a `topic_path` method for the subscriber class in the pubsub project, only `PublisherClient` has it. Perhaps that section of the docs was not updated?